src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -172,9 +172,8 @@ export class InstructorCoursesPageComponent implements OnInit {
     }).subscribe((courseArchive: CourseArchive) => {
       this.loadInstructorCourses();
       if (courseArchive.isArchived) {
-        this.statusMessageService.showSuccessMessage(`The course has been archived.
-          It will not appear in the home page any more. You can access archived courses from the 'Courses' tab.
-          Go there to undo the archiving and bring the course back to the home page.`);
+        this.statusMessageService.showSuccessMessage(`The course ${courseId} has been archived 
+           and will no longer be visible in the home page. It can be accessed through the 'Courses' tab.`);
```

Comment:
> It can be accessed through the 'Courses' tab.

I feel this sentence is unnecessary as the user is still on the same page after archiving the course, so it is rather self-explanatory. 

I think the better approach is to stick to the original message posted in the issue if we do not have a better phrasing:
`The course ${courseId} has been archived. It will not appear on the home page anymore.`